add links to updated quota management docs (#4649)
getsentry/sentry-docs: src/docs/product/accounts/quotas/index.mdx

1. **SDK configuration**

>    The SDK configuration either allows the event or filters the event out. Learn more in [errors](product/accounts/quotas/manage-event-stream-guide/) and [transactions](/product/accounts/quotas/manage-transaction-quota/) quota management docs or in [Filtering Events](/platform-redirect/?next=/configuration/filtering/).

Author: ```suggestion
   The SDK configuration either allows the event or filters the event out. Learn more in [errors](/product/accounts/quotas/manage-event-stream-guide/) and [transactions](/product/accounts/quotas/manage-transaction-quota/) quota management docs or in [Filtering Events](/platform-redirect/?next=/configuration/filtering/).
```

>    The SDK configuration either allows the event or filters the event out. Learn more in [errors](/product/accounts/quotas/manage-event-stream-guide/) and [transactions](/product/accounts/quotas/manage-transaction-quota/) quota management docs or in [Filtering Events](/platform-redirect/?next=/configuration/filtering/).

Author: ```suggestion
   The SDK configuration either allows the event or filters the event out. Learn more in [errors](/product/accounts/quotas/manage-event-stream-guide/) and [transactions](/product/accounts/quotas/manage-transaction-quota/) quota management docs or in the [filtering docs for your specific SDK](/platform-redirect/?next=/configuration/filtering/).
```

- From specific release versions of your code
- From certain IP addresses.

Explore these by navigating to **[Project] > Settings > Inbound Filters**. Commonly-set filters are discussed here for your quick reference. For additional information and examples, see [Inbound Data Filters in our guide to managing your error quota](/product/accounts/quotas/manage-event-stream-guide/#3-inbound-data-filters). Commonly-set filters include:

### IP Filters

> If you have a rogue client, Sentry supports blocking an IP from sending data. Navigate to **[Project] > Settings > Inbound Filters** to add the IP addresses (or subnets) to **Filter errors from these IP addresses**.

Author: ```suggestion
If you have a rogue client, Sentry supports blocking an IP from sending data. Navigate to **[Project] > Settings > Inbound Filters** to add the IP addresses (or subnets) in the "IP addresses" field.
```

### Filter by Release

> <Alert level="info">Filter by release is available on the Business plan.</Alert>

Author: ```suggestion
<Note>

This feature is available only if your organization is on a Business plan.

</Note>
```

> If you discover a problematic release causing excessive noise, Sentry supports ignoring all events from that release. Navigate to **[Project] > Settings > Inbound Filters**, then add the releases to **Filter errors from these releases**.

Author: ```suggestion
If you discover a problematic release causing excessive noise, Sentry supports ignoring all events from that release. Navigate to **[Project] > Settings > Inbound Filters**, then add the releases to the "Releases" field
```

### Filter by Error Message

> <Alert level="info">
>   Filter by error message is available on the Business plan.
> </Alert>

Author: ```suggestion
<Note>

This feature is available only if your organization is on a Business plan.

</Note>
```

> Sentry supports filtering out a specific or certain kind of error as well. Navigate to **[Project] > Settings > Inbound Filters**, then add the error message to **Filter errors by error message**.

Author: ```suggestion
Sentry supports filtering out a specific or certain kind of error as well. Navigate to **[Project] > Settings > Inbound Filters**, then add the error message to the "Error Message" field.
```
